app-android/src/main/java/com/ustadmobile/port/android/view/ReportTemplateFragment.kt

```diff
@@ -0,0 +1,100 @@
+package com.ustadmobile.port.android.view
+
+import android.os.Bundle
+import android.view.*
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.toughra.ustadmobile.R
+import com.toughra.ustadmobile.databinding.ItemReportTemplateListBinding
+import com.ustadmobile.core.controller.ReportTemplatePresenter
+import com.ustadmobile.core.controller.UstadListPresenter
+import com.ustadmobile.core.util.ext.toStringMap
+import com.ustadmobile.core.view.ReportTemplateView
+import com.ustadmobile.lib.db.entities.Report
+import com.ustadmobile.port.android.view.ext.setSelectedIfInList
+import com.ustadmobile.port.android.view.util.ListHeaderRecyclerViewAdapter
+import com.ustadmobile.port.android.view.util.SelectablePagedListAdapter
+
+
+class ReportTemplateFragment(): UstadListViewFragment<Report, Report>(),
```

Comment:
Should be ReportTemplateListFragment (missing List). Same naming issue goes for presenter, view, tests, etc.